package.json

```diff
@@ -34,10 +34,8 @@
     "@aws-sdk/client-cloudformation": "^3.622.0",
     "@types/aws-lambda": "^8.10.142",
     "aws-cdk-lib": "^2.150.0",
-    "cdk-serverless-clamscan": "^2.8.1",
     "constructs": "^10.3.0",
     "esbuild": "^0.21.5",
-    "luxon": "^3.5.0",
```

Comment:
I removed luxon from the project because we have `date-fns` installed in the root. @benjaminpaige lemme know if the choice was intentional and I'll revert my changes

Reply:
Email formatting tests are passing, FWIW